pkg/controller/logstorage/logstorage_controller.go

```diff
@@ -167,12 +171,17 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	// Watch all the secrets created by this controller so we can regenerate any that are deleted
 	for _, secretName := range []string{
 		render.TigeraElasticsearchCertSecret, render.TigeraKibanaCertSecret,
-		render.OIDCSecretName, render.DexObjectName} {
+		render.OIDCSecretName, render.DexObjectName, relasticsearch.PublicCertSecret} {
```

Comment:
Will `relasticsearch.PublicCertSecret` exist in the operator namespace? My current assumption is that it has just the cert portion of TigeraElasticsearchCertSecret, based on previous comments.

Reply:
That is correct, it will contain just the cert potion and will exist in the Operator namespace. From there it will be copied to any namespace that has components that need to communicate with the gateway.